docker/nginx/libs/skynet/account.lua

```diff
@@ -78,7 +87,7 @@ function _M.has_subscription()
 end
 
 function _M.is_auth_required()
-    return os.getenv("ACCOUNTS_LIMIT_ACCESS") == "authenticated"
+    return os.getenv("ACCOUNTS_LIMIT_ACCESS") == "authenticated" or os.getenv("ACCOUNTS_LIMIT_ACCESS") == "subscription"
```

Comment:
maybe store the result of `os.getenv` in a local variable here? not sure whether it caches this value although I guess it does

Reply:
`os.getenv` is a method that returns a single env variable, unless I'm reading these docs incorrectly: https://www.lua.org/manual/5.1/manual.html#pdf-os.getenv